script.js

```diff
@@ -1,24 +1,198 @@
+    languages: ['html', 'css', 'javascript'],
+    liveLink: 'https://www.google.com',
+    sourceLink: 'https://www.google.com'
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    info: {
+      info1: 'CANOPY',
+      info2: 'Back End Dev',
+      info3: 2015
+    },
+    screenshot: 'img/snapshot2.svg',
+    popupScreenshot: 'img/popup_snapshot.svg',
+    mobileScreenshot: 'img/multipoststories.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+    languages: ['html', 'css', 'javascript'],
+    liveLink: 'https://www.google.com/',
+    sourceLink: 'https://www.google.com/'
+  },
+  {
+    id: 3,
+    name: 'Facebook 360',
+    info: {
+      info1: 'CANOPY',
+      info2: 'Back End Dev',
+      info3: 2015
+    },
+    screenshot: 'img/snapshot3.svg',
+    popupScreenshot: 'img/popup_snapshot.svg',
+    mobileScreenshot: 'img/Tonic2.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+    languages: ['html', 'css', 'javascript'],
+    liveLink: 'https://www.google.com/',
+    sourceLink: 'https://www.google.com/'
+  },
```

Comment:
### Your Third Card
![third-project-missing-link](https://user-images.githubusercontent.com/93039359/180131500-e8d7a153-b979-453d-8499-f65691ec97c0.png)

- [ ] Your third card image is broken. :cry: You can fix it :blush: as shown below :arrow_down:

| Your Link | Correct Link |
|-------|--------|
| 'img/**T**onic2.png' | 'img/**t**onic2.png' |